### PR TITLE
chore: update node version

### PR DESCRIPTION
**Pull request by gashcrumb**
This change updates the node version on CI to the latest 16.x LTS and also applies the output of `npm audit fix`.

### CODE REVIEW

**Comment by gashcrumb** [comment]
I see there's a couple deprecation notices in the job output, I'll take a look at those tomorrow as part of this.

**Comment by gashcrumb** [comment]
Okay, this seems ready to go

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -63,7 +63,7 @@ jobs:
 
       # Test run video was always captured, so this action uses "always()" condition
       - name: upload Cypress test run video
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v3
```
Comment (kelvah) [inline .github/workflows/integration.yml:66]: There's also another usage of `actions/upload-artifact@v2` on line 58

Comment (gashcrumb) [reply .github/workflows/integration.yml:66]: ooh, missed that one, thanks!  I've updated that.

**Comment by gashcrumb** [comment]
Hmm, cypress still doesn't seem to get consistently cached I guess, looking into that...

**Comment by gashcrumb** [comment]
Interesting, so 2 caches appear to get created with the same name, after which only the smaller 2nd cache gets loaded and we get a failed build since that smaller cache doesn't have all the files in it.

**Comment by gashcrumb** [comment]
Think this now works, I'd like to manually re-run the jobs a couple times to be sure the cache behaves as it should.

**Comment by gashcrumb** [comment]
Tried a couple re-runs as mentioned in the previous comment.  Seems to work well.  

We could probably stand to de-duplicate some of the action config if possible, but let's save that for a future effort.

**Review by indraraj — APPROVED** [review]
LGTM 👍

**Pull request merged**